wallet/src/repository/wallet/state.rs

```diff
@@ -70,10 +70,12 @@ pub struct State {
     pub transaction_next_id: u32,
     /// Current UTXO set (including pending movements)
     pub utxo_set: model::UtxoSet,
+    /// Tagged used UTXO set
```

Comment:
I think `used` is in conflict with the `U` in `Utxo` standing for `unspent` :stuck_out_tongue_winking_eye: 

Maybe we can simply name this `used_outputs` and `model::UsedOutputs`.